commands/function-stats.md

```diff
@@ -1,3 +1,15 @@
-Work in progress.
+Return information about the function that's currently running.
 
-Please refer to [the Redis Functions issue](https://github.com/redis/redis/pull/9780) and related pull requests for the time being.
\ No newline at end of file
+The reply consists of the following information about the running script:
+
+* **name:** the name of the function.
+* **command:** the command and arguments used for invoking the function.
+* **duration_ms:** the function's runtime duration in milliseconds.
+
+If there's no in-flight function, the server replies with a _nil_.
+
+In addition, the reply also includes returns the list of the function execution engines that are available.
```

Comment:
We can add more information, What extra information you expect? (memory usage? number of libraries?)

Reply:
don't know.. currently it is a place holder of per-engine stats with no actual data.
didn't we used to have something there?
in INFO we added separate memory metrics for Eval and Functions, and also a metric for the Lua VM (`used_memory_vm_eval`), so if we add a memory metric here, it could be a drill down of per-vm metric that's just for functions (excludes EVAL).

at the very list, maybe add a metric of the number of functions or number of libraries?

Reply:
p.s.i suppose the metrics themselves don't have to be documented here.
i don't wanna run here to update the docs every time we add a metric (maybe just for the important ones).
so we should just make sure this docs page lists the details correctly (assuming there are some per engine metrics.
or maybe give the number of functions as an example.

Reply:
@MeirShpilraien can you please add a metric of the number of functions in each engine?
(and update this PR with that too)

my aim is to make it clear that this command has per-engine statistics, so i want at least one stat.